sass/layout/_queries.scss

```diff
@@ -69,16 +76,16 @@ $breakpoints: () !default;
   $source: $breakpoints,
   $scale: true
 ) {
-  @if ($scale) and variable-exists('sizes') and (type-of($sizes) == 'map') {
-    $source: map-merge($sizes, $source);
+  @if ($scale) and meta.variable-exists('sizes') and (meta.type-of($sizes) == 'map') {
+    $source: map.merge($sizes, $source);
   }
 
   // get breakpoint
-  $break: get-token($source, $break, $do);
+  $break: tokens.get($source, $break, $do);
 
   // validate
-  @if (type-of($break) != 'number') or unitless($break) {
-    @return _a_error(
+  @if (meta.type-of($break) != 'number') or math.is-unitless($break) {
```

Comment:
I was getting an error with math.unitless. Is there something other than this `is-unitless` I should be using here?